org.onebusaway.iphone.xcodeproj/project.pbxproj

```diff
@@ -2474,44 +2474,22 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 2090FA85E96D4CC26E750866 /* Pods-OneBusAway.debug.xcconfig */;
 			buildSettings = {
-				ALWAYS_SEARCH_USER_PATHS = NO;
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```

Comment:
I think this line should be removed. I believe this makes Cocoapods grumble on `pod install`